config/site_settings.yml

```diff
@@ -1202,6 +1202,7 @@ email:
   max_emails_per_day_per_user: 100
   enable_staged_users: true
   maximum_staged_users_per_email: 10
+  maximum_recipients_per_new_group_email: 10
```

Comment:
Do we need to add a minimum and maximum values for this setting?

Reply:
I believe a minimum value can be `1`. I do not know what is an appropriate value for the maximum value because I believe some people use group emails with many staged users (for example, support forums).